R/reporter-progress.R

```diff
@@ -258,6 +262,10 @@ ProgressReporter <- R6::R6Class(
     },
 
     end_reporter = function() {
+      if (self$is_full()) {
+        return()
+      }
+
```

Comment:
Why is this needed here? I think if we are at the end of the test suite, then no need to quit without the report.